drivers/filesystems/btrfs/btrfs.h

```diff
@@ -9,12 +9,14 @@
 #pragma once
 
 #include <stdint.h>
+#include <assert.h>
```

Comment:
Is that (--> using that header) allowed in kernel mode?

Reply:
it seems I forgot to surround it with ifdef like the static_assert at line 306

Reply:
done in e720186ed8beb765c75222249dfe7e6d78bb3ab8